src/Forms/Prism.Forms/Services/Dialogs/DialogResult.cs

```diff
@@ -5,9 +5,20 @@
 
 namespace Prism.Services.Dialogs
 {
+    /// <summary>
+    /// An <see cref="IDialogResult"/> that contains <see cref="IDialogParameters"/> from the dialog
+    /// and the <see cref="System.Exception"/> of the dialog.
```

Comment:
you can use the inherit doc tag instead of duplicating

Reply:
I use the inherit doc tag.  
https://github.com/PrismLibrary/Prism/pull/2761/files#diff-0d0a9b1f968cdc60f0420e45398c06795f2b4d473269f729f6a2f977c96ac071